### PR TITLE
Add more information about KDE 6

### PR DESCRIPTION
**Pull request by AaronDewes**
Related to https://github.com/microsoft/vscode/pull/187402, this makes the necessary docs change.

This can be merged before that PR is merged, it informs users about the compatibility issue with KDE 6, which that PR will fix.

The neon-specific instructions will only apply after the PR is merged, but I thought it would make sense to already add them so users can take these steps in advance.

### CODE REVIEW

File: docs/editor/settings-sync.md [hunk 1]
```diff
@@ -177,8 +177,19 @@ If the error you're seeing is "Cannot create an item in a locked collection", ch
...
 
+On KDE Neon Unstable, `kwallet5` and `kwallet6` currently conflict with each other. It is recommended to remove kwallet5 and reinstall kwallet6 to fix any potential issues:
+
+```
+apt remove libkf5wallet-bin
+apt install --reinstall kf6-kwallet
+```
+
+After a reboot, you will now be running KWallet6 properly.
+
```
[the remark this answers is not in the document]
Comment (AaronDewes) [reply docs/editor/settings-sync.md:192]: Okay, I'll remove it then :+1:.

**Review by TylerLeonhardt — COMMENTED** [review]
Thanks for contributing! I left a comment... one thing I would like to hear you try is to set `"password-store"` to the value of `"gnome"`... this will attempt to use libsecret to see if there's a compatible keyring and AFAIK, kwallet supports libsecret...... so I wonder if we get KDE6 for "free" leveraging that.

**Comment by AaronDewes** [comment]
> one thing I would like to hear you try is to set `"password-store"` to the value of `"gnome"`... this will attempt to use libsecret to see if there's a compatible keyring and AFAIK, kwallet supports libsecret...... so I wonder if we get KDE6 for "free" leveraging that.


I'll try in a minute. But updating Electron will probably happen long-term anyway, but this could be added to the docs as a workaround.

**Comment by AaronDewes** [comment]
Updated docs with a slightly modified version of your workaround, thanks!

**Review by TylerLeonhardt — APPROVED** [review]

**Pull request merged**